contentcuration/contentcuration/serializers.py

```diff
@@ -600,7 +605,8 @@ class Meta:
         fields = (
             'id', 'created', 'updated', 'name', 'description', 'has_changed', 'editors', 'main_tree', 'trash_tree',
             'staging_tree', 'source_id', 'source_domain', 'ricecooker_version', 'thumbnail', 'version', 'deleted',
-            'public', 'thumbnail_url','thumbnail_encoding', 'pending_editors', 'viewers', 'tags', 'preferences')
+            'public', 'thumbnail_url','thumbnail_encoding', 'pending_editors', 'viewers', 'tags', 'preferences',
+            'secret_tokens', 'primary_token')
```

Comment:
Does secret_tokens get used in the front-end? If not, can probably just remove for now